### PR TITLE
Add link to Alarm Clock

### PR DESCRIPTION
**Pull request by DavisNT**
Add link to [Mopidy-AlarmClock](https://github.com/DavisNT/mopidy-alarmclock) if it is enabled in Mopidy config (e.g. installed and loaded).

Related to https://github.com/woutervanwijk/Pi-MusicBox/issues/270

### CODE REVIEW

File: mopidy_musicbox_webclient/web.py [hunk 1]
```diff
@@ -41,10 +41,17 @@ def initialize(self, config, path):
                                'using %s', port)
             ws_url = "ws://%s:%d/mopidy/ws" % (host, port)
 
+        alarmclock = False
+        try:
+            alarmclock = config['alarmclock']['enabled']
```
Comment (kingosticks) [inline mopidy_musicbox_webclient/web.py]: A try block for this isn't really necessary.

**Comment by DavisNT** [comment]
@kingosticks I removed try/except. I hope there won't be any cases when corrupted config will have non-dictionary types (and crash webclient).

**Comment by kingosticks** [comment]
I think you can probably use get() with a default value, or whatever the
dictionary method is called. Mopidy sanitises the config anyway.

**Comment by DavisNT** [comment]
If Mopidy always sanitizes the config (`config['alarmclock']` is always dictionary if present) then this code should work OK. I think that adding `get()` won't make code much shorter, but will add something that is not used in other places. Is current code OK to be merged?

**Comment by kingosticks** [comment]
I think that's there now is nice and clear so that's fine by me.  

In the diff it looks like settings and shutdown now have slightly different ui-block-a and -b but it's hard to tell what's going on my phone and I can't remember the difference. But assuming that's nothing then I'll merge when I get home later.

**Comment by DavisNT** [comment]
`ui-block-a` and `ui-block-b` determine alignment (left or right) if two columns are displayed. These classes are changed back in `gui.js` line 528 if Alarm Clock link is hidden.

**Comment by kingosticks** [comment]
Oh yeh right those button things, thanks for the hint.

**Pull request merged**